internal/plugins/helm/flags.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package helm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/pflag"
+
+	gencrd "github.com/operator-framework/operator-sdk/internal/generate/crd"
+	sdkscaffold "github.com/operator-framework/operator-sdk/internal/scaffold"
+)
+
+const (
+	apiVersion       = "api-version"
+	kind             = "kind"
+	crdVersion       = "crd-version"
+	helmChart        = "helm-chart"
+	helmChartVersion = "helm-chart-version"
+	helmChartRepo    = "helm-chart-repo"
+)
+
+type APIFlags struct {
```

Comment:
These are only ever used in one place, so why not just have these fields in `createAPIPlugin`? At the very least this shouldn't be exported.

Reply:
We split the PR init/api. The flags are common for both and are used for them. See: #3239